.circleci/config.yml

```diff
@@ -196,6 +177,68 @@ jobs:
           path: build/test-results
       - store_artifacts:
           path: build/reports/tests
+  cas1_e2e_tests:
+    docker:
+      - image: mcr.microsoft.com/playwright:v1.34.1-focal
+    circleci_ip_ranges: true # opt-in to jobs running on a restricted set of IPs
+    steps:
+      - run:
+          name: Clone E2E repo
+          command: |
+            git clone https://github.com/ministryofjustice/hmpps-approved-premises-e2e.git .
+      - run:
+          name: Update npm
+          command: 'npm install -g npm@latest'
+      - node/install-packages
+      - run:
+          name: E2E Check
+          command: |
+            npm run test
+      - store_artifacts:
+          path: playwright-report
+          destination: playwright-report
+      - store_artifacts:
+          path: test-results
+          destination: test-results
+      - slack/notify:
+          event: fail
+          channel: << pipeline.parameters.alerts-slack-channel >>
+          template: basic_fail_1
+  cas3_e2e_tests:
+    parallelism: 4
```

Comment:
Noticed CAS1 doesn't have this and it sounds useful to keep deploys faster. What's the max? Guessing it's the number of cores on whatever sized resource we pay for and so can't be increased?

Reply:
Yeah, I haven't bothered for CAS1, as we have fewer e2e tests and they're only split across two files. We could try increasing it, but it'll be diminishing returns.